content/events-api/subscription.md

```diff
@@ -76,3 +77,32 @@ This request is JSON-based (`Content-Type: application/json`) and contains wrapp
 | `pim_source` | Identify which Akeneo PIM sent the event. | `demo.akeneo.com` | From `AKENEO_PIM_URL` environment variable |
 | `data` | Contains the resource data. Learn more about the data wrapper, including its JSON schema, on our [Events API Reference](/events-reference/events-reference-serenity/products.html). | | |
 
+
+## Debugging events
+
+![Download logs button](../img/events-api/connection-download-logs-button.png)
+
+### Log types
+As you can read on our Help center page [Subscribe and manage **events**](https://help.akeneo.com/pim/serenity/articles/manage-event-subscription.html#debug), the log file gives you access to 4 log levels ang 5 log types: 
+- `ERROR The endpoint returned an error.` In that case, the connected app received the event request, but something went wrong, and it answered with an error. 
+- `ERROR The endpoint failed to answer under 500 ms.` This error means that the connected application did not answer quickly enough. 
+- `WARNING The maximum number of requests per hour has been reached.` If you have this warning log, you could be interested in this article's limit and scalability paragraph. ;) 
+- `NOTICE The event was not sent because it was raised by the same connection.` In that case, your connected application raised an event in the PIM through the REST API, so we chose not to send this event to avoid an endless loop between the PIM and your app.
+- `NOTICE The event was not sent because the product does not exist or the connection does not have the required permissions.` When the PIM doesn't send an event because of a lack of permission, it can be normal. You set up the connection permission to not receive events on products that don't interest your app. If you think you should have received this event, please look at our [permission configuration paragraph](https://help.akeneo.com/pim/serenity/articles/manage-event-subscription.html#manage-your-permissions). 
```

Comment:
```suggestion
- `NOTICE The event was not sent because the product does not exist or the connection does not have the required permissions.` When the PIM doesn't send an event because of a lack of permission, it can be normal. For instance, if you previously set up the connection permission to not receive events on products that are irrelevant for this particular app. If you think you should have received this event, please look at our [permission configuration section](https://help.akeneo.com/pim/serenity/articles/manage-event-subscription.html#manage-your-permissions). 
```